src/MPAGSCipher/RunCaesarCipher.hpp

```diff
@@ -1,10 +1,9 @@
 #ifndef MPAGSCIPHER_RUNCAESARCIPHER_HPP
 #define MPAGSCIPHER_RUNCAESARCIPHER_HPP
-
 #include <string>
+using namespace std;
+
 
-//! Encrypt or decrypt text using the Caesar cipher with the given key
-std::string runCaesarCipher(const std::string& inputText, const std::size_t key,
-                            const bool encrypt);
+string runCaesarCipher(const string& inputText, const size_t key, const bool encrypt, const bool decrypt);
```

Comment:
Instead of adding the `decrypt` parameter, it is more convenient to use `enum` here:
```c++
enum class CipherMode{
    Encrypt,
    Decrypt
};
```
This is more descriptive and requires only one parameter (i.e. does not increase the complexity of `runCaesarCipher`).